Makefile

```diff
@@ -2,6 +2,8 @@ TERRAFILE_VERSION=0.8
 ARM_TEMPLATE_TAG=1.1.0
 RG_TAGS={"Product" : "Database of Qualified Teachers"}
 SERVICE_SHORT=ccbl
+cluster_rg=""
```

Comment:
I believe these are not required anymore

Reply:
removed unused